.github/workflows/e2e-tests-preview.yml

```
@@ -0,0 +1,465 @@
+              INSTALL_BROWSER="$BROWSER"
+              echo "🖥️ Desktop browser: $INSTALL_BROWSER"
+              ;;
+            *)
+              echo "❌ Unknown browser type: $BROWSER"
+              exit 1
+              ;;
+          esac
+
+          # Always verify browser installation for reliability
+          echo "🔍 Verifying Playwright browser: $INSTALL_BROWSER..."
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)